src/mscorlib/src/System/Runtime/CompilerServices/AsyncMethodBuilder.cs

```diff
@@ -721,10 +699,6 @@ internal void SetNotificationForWaitCompletion(bool enabled)
         [MethodImpl(MethodImplOptions.AggressiveInlining)] // method looks long, but for a given TResult it results in a relatively small amount of asm
         private Task<TResult> GetTaskForResult(TResult result)
```

Comment:
Needs to change to `internal static` to not clash with https://github.com/dotnet/coreclr/pull/13907

Reply:
nvm, merge will catch it

Reply:
fixed the conflict